### DISCUSSION TITLE
Questions about the performance comparison

### DISCUSSION THREAD

**Discussion by AmbitionXiang** [edgelesssys/constellation#3339] [Q&A]
Hi~ I noticed that Constellation performs even [faster than insecure AKS and GKS](https://constellation-docs.netlify.app/constellation/2.4/overview/performance) in terms of API latency.  It is amazing! How did you do that? Why is it even faster than k8s without TEEs?

**Comment by 3u13r** [comment]
duplicate of https://github.com/edgelesssys/constellation/discussions/3345
